apps/ccf-eui/src/app/app.component.html

```diff
@@ -220,7 +220,37 @@
       <span>Test</span>
     </mat-menu>
   </hra-expansion-panel-actions>
-  <hra-expansion-panel-header-content> </hra-expansion-panel-header-content>
+  <hra-expansion-panel-header-content>
+    <form>
```

Comment:
Please use a form control instead of `ngModel` etc. We can work together to get it working correctly. Also we might want to move this organ selection into it's own component.